mashup-example/using-build-tool/qlik-sense-saas-platform-sdk/configure.js

```diff
@@ -5,7 +5,7 @@ const n = embed.createConfiguration({
   context: {
     theme: 'light',
     language: 'en-US',
-    constraints: {
+    interactions: {
       active: false, // do not allow interactions
```

Comment:
according to the comment this should have been set to true. With inverting the booleans it should actually be correct.